token/services/ttx/accept.go

```diff
@@ -51,18 +64,25 @@ func (s *acceptView) Call(context view.Context) (interface{}, error) {
 	if logger.IsEnabledFor(zapcore.DebugLevel) {
 		logger.Debugf("send back ack")
 	}
+
+	var sigma []byte
+	logger.Debugf("signing ack response: %s", hash.Hashable(rawRequest))
+	signer, err := view2.GetSigService(context).GetSigner(view2.GetIdentityProvider(context).DefaultIdentity())
+	if err != nil {
+		return nil, errors.WithMessagef(err, "failed to get signer for default identity")
+	}
+	sigma, err = signer.Sign(rawRequest)
+	if err != nil {
+		return nil, errors.WithMessage(err, "failed to sign ack response")
+	}
 	// Ack for distribution
+	// Send the signature back
 	session := context.Session()
-	// Send the proposal response back
-	err = session.Send([]byte("ack"))
-	if err != nil {
-		return nil, err
+	logger.Debugf("ack response: [%s] from [%s]", hash.Hashable(sigma), view2.GetIdentityProvider(context).DefaultIdentity())
```

Comment:
can we check first if debug log is enabled using `if logger.IsEnabledFor(zapcore.DebugLevel) {` ?
This should improve performance when debug is disbaled